### ISSUE TITLE
NVDA should have a browsing in loop function. The function should anounce when getting/returning to the end/top of a web page.

### ISSUE THREAD

**Issue by FDX-VQ** [nvaccess/nvda#11872]

**Comment by leonardder** [comment]
@FDX-VQ: Please read the information [here](https://github.com/nvaccess/nvda/wiki/Github-issue-template-explanation-and-examples) before filing any additional issues. Closing as invalid.
